library/src/test/java/com/novoda/downloadmanager/MigrationExtractorTest.java

```diff
@@ -48,21 +48,21 @@
             .build();
 
     private final SqlDatabaseWrapper database = mock(SqlDatabaseWrapper.class);
-    private final InternalFilePersistence internalFilePersistence = mock(InternalFilePersistence.class);
+    private final FilePersistence filePersistence = mock(FilePersistence.class);
 
     private MigrationExtractor migrationExtractor;
 
     @Before
     public void setUp() {
-        migrationExtractor = new MigrationExtractor(database, internalFilePersistence, "base");
+        migrationExtractor = new MigrationExtractor(TestStorageRootFactory.create(), database, filePersistence, "base");
```

Comment:
seems like we use `TestStorageRootFactory.create()` in quite few places. Should we extract it to be final field ?